### DISCUSSION TITLE
How to specify the where clause in the findOne API method

### DISCUSSION THREAD

**Discussion by pmobbe** [nocodb/nocodb#344] [Q&A]
I am trying NocoDB out and I have a Test table that Im trying to query via the API. I am having some issues with the syntax of the where clause and I cant find any documentation. 

I want to find a record where the date field is equal to a certain date. Since I am using postgres I was wanting to use some of its date operators to do this. 

I have tried: `where=(dateField::date,eq,'some date"::date)`

NocoDB doesn't like this and is saying the syntax is incorrect. 

Any ideas of how to achieve the above but with proper syntax? 

Thanks,

**Comment by pXcje** [comment]
I would expect the double qoutes are not correct.
You will find a list of where examples at [(https://github.com/o1lab/xmysql)]

**Comment by wingkwong** [comment]
Documentation : [REST APIs](https://docs.nocodb.com/developer-resources/rest-apis) / [GraphQL APIs](https://docs.nocodb.com/developer-resources/graphql-apis)
